src/app/designationEditor/personalOptionsModal/personalOptionsModal.tpl.html

```diff
@@ -1,17 +1,23 @@
 <div class="modal-body">
   <form name="personalForm">
     <!-- Nav tabs -->
-    <ul class="nav nav-tabs" ng-init="$ctrl.tab = 'givingLinks'">
+    <ul class="nav nav-tabs">
       <li ng-class="{'active': $ctrl.tab === 'givingLinks'}">
-        <a href="" ng-click="$ctrl.tab = 'givingLinks'">
+        <a href="" ng-click="$ctrl.tab = 'newsletter'">
+          MPDX Newsletter <span class="xs-hide">Form</span>
+        </a>
+        <a href="" ng-click="$ctrl.tab = 'givingLinks'"
+           ng-if="$ctrl.designationType === 'National Staff'">
```

Comment:
It looks like this got pushed back to National Staff only, maybe a rebase error? Or was the decision switched back?

Reply:
This was based off that branch before the change. It'll probably have a conflict on merge. I'll fix it here anyway.

Reply:
Actually. I don't think that ever got changed.